app/src/main/java/com/infomaniak/swisstransfer/ui/images/illus/onboarding/StorageBoxPileDark.kt

```diff
@@ -0,0 +1,632 @@
+                    curveToRelative(-0.69f, -0.32f, -1.26f, -0.63f, -1.62f, -0.94f)
+                    lineToRelative(-1.28f, 1.53f)
+                    curveToRelative(0.58f, 0.48f, 1.34f, 0.88f, 2.07f, 1.22f)
+                    curveToRelative(0.71f, 0.33f, 1.53f, 0.66f, 2.12f, 0.92f)
+                    close()
+                    moveTo(99.24f, 93.28f)
+                    curveToRelative(-1.94f, -1.63f, -2.24f, -3.67f, -1.6f, -5.16f)
+                    curveToRelative(0.63f, -1.47f, 2.27f, -2.61f, 4.76f, -2.21f)
+                    lineToRelative(0.31f, -1.97f)
+                    curveToRelative(-3.31f, -0.52f, -5.89f, 1.02f, -6.91f, 3.4f)
+                    curveToRelative(-1.01f, 2.37f, -0.4f, 5.34f, 2.15f, 7.48f)
+                    close()
+                    moveTo(102.4f, 85.91f)
+                    curveToRelative(0.4f, 0.06f, 1.02f, 0.28f, 1.77f, 0.59f)
+                    curveToRelative(0.74f, 0.31f, 1.53f, 0.69f, 2.24f, 1.05f)
+                    lineToRelative(0.89f, -1.79f)
+                    arcToRelative(45.0f, 45.0f, 0.0f, false, false, -2.35f, -1.1f)
+                    curveToRelative(-0.77f, -0.32f, -1.58f, -0.62f, -2.24f, -0.73f)
+                    close()
+                    moveTo(107.44f, 85.84f)
+                    curveToRelative(-0.56f, -0.4f, -1.58f, -0.94f, -2.55f, -1.4f)
+                    curveToRelative(-0.95f, -0.45f, -2.02f, -0.9f, -2.65f, -1.04f)
+                    lineToRelative(-0.43f, 1.95f)
+                    curveToRelative(0.39f, 0.09f, 1.26f, 0.44f, 2.23f, 0.89f)
+                    curveToRelative(0.95f, 0.45f, 1.84f, 0.93f, 2.25f, 1.22f)
+                    close()
+                    moveTo(102.24f, 83.4f)
+                    curveToRelative(-2.85f, -0.63f, -5.71f, 0.76f, -7.04f, 3.06f)
+                    arcToRelative(5.65f, 5.65f, 0.0f, false, false, -0.63f, 4.06f)
+                    curveToRelative(0.32f, 1.48f, 1.21f, 2.94f, 2.76f, 4.29f)
+                    lineToRelative(1.31f, -1.51f)
+                    curveToRelative(-1.29f, -1.13f, -1.91f, -2.23f, -2.12f, -3.21f)
+                    arcToRelative(3.66f, 3.66f, 0.0f, false, true, 0.41f, -2.64f)
+                    curveToRelative(0.9f, -1.57f, 2.9f, -2.55f, 4.87f, -2.11f)
+                    close()
+                }
+                path(
+                    fill = SolidColor(Color(0xFFA98116)),
+                    stroke = null,
+                    strokeLineWidth = 0.0f,
+                    strokeLineCap = Butt,
+                    strokeLineJoin = Miter,
+                    strokeLineMiter = 4.0f,
+                    pathFillType = EvenOdd
+                ) {
+                    moveTo(113.15f, 103.25f)
+                    curveToRelative(-0.14f, -0.05f, -10.5f, 3.16f, -7.58f, 7.85f)
+                    curveToRelative(-0.47f, -2.07f, 0.5f, -4.47f, 7.58f, -7.85f)
+                }
+            }
+            group {
```

Comment:
This group looks like it can be removed.

Reply:
About groups in generaé, I have answered to the same comment here: https://github.com/Infomaniak/android-SwissTransfer/pull/222#discussion_r1870905083